config/application.rb

```diff
@@ -36,5 +36,8 @@ class Application < Rails::Application
 
     config.x.disable_password = ENV['DISABLE_PASSWORD'] == 'true'
     config.x.app_password = ENV['SCHEMA_REGISTRY_PASSWORD'] || 'avro'
+
+    config.x.allow_response_caching = ENV['ALLOW_RESPONSE_CACHING'] == 'true'
+    config.x.cache_max_age = (ENV['CACHE_MAX_AGE'] || 2592000).to_i
```

Comment:
Maybe we should be defensive if someone doesn't type an integer here (e.g. 'wxyz')?